untwine/Common.cpp

```diff
@@ -55,7 +55,11 @@ MapContext mapFile(const std::string& filename, bool readOnly, size_t pos, size_
 #ifndef _WIN32
     ctx.m_fd = ::open(filename.data(), readOnly ? O_RDONLY : O_RDWR);
 #else
+#ifdef _MSC_VER
```

Comment:
I'd appreciate some comments saying what cases are which. If you do MinGW all day, you may know what's what, but for everyone else, these cases are unclear.